active_record_stats.gemspec

```
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency 'activerecord', '>= 4.0'
-  spec.add_runtime_dependency 'statsd-instrument', '~> 2.0', '>= 2.0.4'
+  spec.add_runtime_dependency 'statsd-instrument', '~> 3.0'
```

keep it free form, so we don't have to keep updating in the future:

```suggestion
  spec.add_runtime_dependency 'statsd-instrument', '>= 2.0.4'
```